PantryNodeReact/src/Components/Feed/Feed.tsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react'
+import { Box, Typography } from '@mui/material';
+
+type FeedProps = {
+    sortedFeedList: { 
+        id: number; 
+        item: string; 
+        expiry_date: string;
+        added_date: string; 
+        quantity: number 
+    }[];
+    
+    sortedExpiredFeedList: { 
+        id: number; 
+        item: string; 
+        expiry_date: string;
+        added_date: string; 
+        quantity: number 
+    }[];
+  };
+  
+
+const Feed = ({ sortedFeedList, sortedExpiredFeedList }: FeedProps) => {
+  return (
+    <Box sx={{ width: '100%', bgcolor: '', p: 2 }}>
+        <Typography variant="h4" gutterBottom style={{ textAlign: "center" }}>Feed</Typography>
+        <div style={{ display: "flex", flexWrap:"wrap" }}>
+
+            {/* Expiring feed list section */}
+            <div style={{ width: "50%" }}>
+                <Typography variant="h5" gutterBottom style={{ textAlign: "center" }}>Expiring Soon Items </Typography>
+
+                {
+                    sortedFeedList.filter((item) => {
+                        const expiryDate = new Date(item.expiry_date).getTime();
+                        const currentDate = new Date().getTime();
+                        const isExpiring = expiryDate >= currentDate;
+                       if(isExpiring) return item;
+                       return null;
```

Comment:
We can also just return isExpiring and remove the return null.

Reply:
correct